packages/framework/src/Twig/ImageExtension.php

```diff
@@ -210,6 +210,14 @@ protected function getImageHtmlByEntityName(array $attributes, $entityName, $add
         $htmlAttributes = $attributes;
         unset($htmlAttributes['type'], $htmlAttributes['size']);
 
+        $useLazyLoading = array_key_exists('lazy', $attributes) ? (bool)$attributes['lazy'] : true;
```

Comment:
Do we want every image to use lazy loading by default ? So every image in administration (e.g. icons) will be lazy loaded as well. Is it desired behavior ?

Reply:
We have only two posibilities. One default disabled, other one enabled. For better performance I choosed default enabled. Icons are actually made by svg font. And I hope even this will be changed. Go go to better performance.

Reply:
Fine by me